app/controllers/users_controller.rb

```diff
@@ -23,7 +23,7 @@ def show
 
   # rubocop: disable Metrics/MethodLength
   def create
-    @user = User.find_by email: user_params[:email].downcase
+    @user = User.find_by(email: user_params[:email].downcase)
```

Comment:
That's fine. We should probably make sure we retain email case in the future, but that's a separate task.